### PR TITLE
ref #92: Add default logging config

### PR DESCRIPTION
**Pull request by umulmrum**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | #92
| License       | MIT

### CODE REVIEW

File: app/config/config_dev.yml [hunk 1]
```diff
@@ -19,4 +19,19 @@ parameters:
...
+    #firephp:
+    #    type: firephp
+    #    level: info
+    #chromephp:
+    #    type: chromephp
+    #    level: info
+    console:
+      type: console
+      process_psr_3_messages: false
+      channels: ["!event", "!doctrine", "!console"]
```
Comment (UlrichKu) [inline app/config/config_dev.yml]: not console?

File: app/config/config_prod.yml [hunk 2]
```diff
@@ -9,3 +9,29 @@ parameters:
   chameleon_system_core.cache.memcache_use_fallback: false
   chameleon_system_core.cache.include_cache_delete_trace_info: false
   chameleon_system_core.debug.show_view_source_html_hints: false
+
+monolog:
+  handlers:
+    main:
+      type: fingers_crossed
```
Comment (UlrichKu) [inline app/config/config_prod.yml]: Does this still fit the upgrade doc with "fingers_crossed is not used anymore"?

File: app/config/config_dev.yml (hunk 1 above)
Comment (umulmrum) [reply app/config/config_dev.yml]: Looks a bit strange that the console logger doesn't log the console channel, but it's just the same term used for different things: The console channel is used for the Symfony Console component and has nothing to do with the log handler.

File: app/config/config_prod.yml (hunk 2 above)
Comment (umulmrum) [reply app/config/config_prod.yml]: Removed the note in the upgrade doc.

**Pull request merged**